### PR TITLE
Feature/mobile

### PR DESCRIPTION
**Pull request by mluena**

### CODE REVIEW

**Comment by davidsingal** [comment]
@mluena please could you add some comments about your progress?

**Comment by mluena** [comment]
Layout it´s done (I still need to implement functionality to check if we are in mobile or not). I am currently working on map view.

**Comment by davidsingal** [comment]
I have the next error, I know the branch is still on WIP.

```
@import '../../../../styles/vars';
^
      File to import not found or unreadable: ../../../../styles/vars
```

File: src/components/widget/component.js [hunk 1]
```diff
@@ -103,6 +109,7 @@ class Widget extends PureComponent {
               isActive={isActive}
               onClick={this.activeToggleHandler}
             >
+            {console.log(isActive)}
```
Comment (migh) [inline src/components/widget/component.js]: I think this should not be commited.

File: src/components/map/index.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ import Component from './component';
 const mapStateToProps = state => ({
   ...state.map,
   mapStyle: mapStyle(state),
-  mapboxApiAccessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN
+  mapboxApiAccessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
+  isCollapse: state.layers.isCollapse
```
Comment (migh) [inline src/components/map/index.js]: the one that exists in layers store is `isCollapsed`, the first time is set to flase but trigger some error...

**Pull request merged**